packages/reporter/cypress/e2e/unit/command_model.cy.ts

```diff
@@ -44,6 +45,15 @@ describe('Command model', () => {
       command = new CommandModel(commandProps({ visible: undefined }))
       expect(command.visible).to.be.true
     })
+
+    it('sets visible to true when visibility check is skipped due to large element set', () => {
+      command = new CommandModel(commandProps({
+        visible: undefined,
+        numElements: MAX_VISIBILITY_CHECK_ELEMENTS,
+      }))
+
+      expect(command.visible).to.be.true
+    })
```

Comment:
Is this written backwards? Or the title is backwards? It shows the not visible icon when visibility check is skipped.
<img width="250" height="139" alt="Screenshot 2025-11-13 at 3 50 45 PM" src="https://github.com/user-attachments/assets/ca4edd19-30a2-4efb-8c73-5e09233e3d4c" />